### ISSUE TITLE
cohesin loading

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#12422]
Could the term "cohesin localization to chromatin"

be renamed
"mitotic cohesin loading"  (this fits the current annotations)
and moved, to be a 'part-of' descendant of "establishment of mitotic sister chromatid cohesion"
(the loading is performed  in S-phase by a 'cohesin loading complex' GO:0032116 SMC loading complex

**Comment by tberardini** [comment]
http://amigo.geneontology.org/amigo/term/GO:0071921

For reference.

**Comment by tberardini** [comment]
Update definition? If yes, please provide updated def.

Is the move to 'part-of' descendant of "establishment of mitotic sister chromatid cohesion" supposed to replace the current 'part-of' relationship to ' sister chromatid cohesion'?

Current stanza:

[Term]
id: GO:0071921
name: cohesin localization to chromatin
namespace: biological_process
def: "A process in which a cohesin complex is transported to, or maintained at, a part of a chromosome that is organized into chromatin." [GOC:mah, PMID:10882066, PMID:17113138]
synonym: "cohesin association with chromatin" EXACT [GOC:lb]
synonym: "cohesin localisation to chromatin" EXACT [GOC:mah]
is_a: GO:0022402 {is_inferred="true"} ! cell cycle process
is_a: GO:0071168 ! protein localization to chromatin
is_a: GO:1902580 {is_inferred="true"} ! single-organism cellular localization
relationship: part_of GO:0007062 ! sister chromatid cohesion
created_by: midori
creation_date: 2010-09-28T11:07:26Z

**Comment by ValWood** [comment]
So, I  had made some notes yesterday to request this change from another paper, which is good!

But, I noticed another small thing.

The current term is not 'mitosis' specific, so It would need to be 

"cohesion loading" 
rather than "mitotic cohesion loading"


This is the process we are trying to capture with this GO term.
https://en.wikipedia.org/wiki/Establishment_of_sister_chromatid_cohesion
Cohesin loading is the "association of cohesin with chromosomes' 
I think the current def is OK, and the current term name can be an exact synonym.

If it turns out that "loading' has a narrower meaning, we can look at this later?

It could have an enabled by, but this is a 'grouping term' for all loading complexes....so it might not work?
http://www.pombase.org/spombe/related/GO:0032116

**Comment by ValWood** [comment]
Is the move to 'part-of' descendant of "establishment of mitotic sister chromatid cohesion" supposed to replace the current 'part-of' relationship to ' sister chromatid cohesion'?

yep.

Let me know if I didn't answer your questions.

Loading cohesin is def part of establishment. Existing link would be redundant.

**Comment by tberardini** [comment]
You said: "The current term is not 'mitosis' specific, ..."

Val, if the term is not 'mitosis' specific, then I can't move the term to be part of '[]mitotic sister chromatid cohesion', it would have to stay where it is.

Given all this, the only edit left to make is rename

'cohesin localization to chromatin'

to

'cohesin loading' (note spelling, not 'cohesion loading' unless that's really what you want)

and retain 'cohesin localization to chromatin' as a (related?) synonym.

Please review and if that's correct, I'll make that change.

**Comment by ValWood** [comment]
Hi Tanya,

Apologies for not being explicit. The current term should now move to be part of
"GO:0034085 establishment of sister chromatid cohesion"

and be renamed  "cohesin loading" (biologist friendly term name)

Once that is in place I can do mitotic terms by Term Genie.

Tks,

Val

**Comment by tberardini** [comment]
Hi Val,

It's nice to have everything nice and clear so that I don't mess anything up. When it's in an area of biology that I'm not so familiar with, I want to be extra sure of the changes.

I updated names and synonyms for the BP term and its regulation children (there were two).  Moved part_of parentage as requested.  Checked resulting relationships of all three terms to make sure they were consistent.

Closing!

Tanya
